nodes/stampzilla-server/handlers/securewebsockethandler.go

```diff
@@ -86,21 +91,21 @@ func sliceHas(s []string, val string) bool {
 	return false
 }
 
-func (wsh *secureWebsocketHandler) Message(s interfaces.MelodySession, msg *models.Message) error {
+func (wsh *secureWebsocketHandler) Message(s interfaces.MelodySession, msg *models.Message) (error, json.RawMessage) {
```

Comment:
Is the second argument in use anywhere? I checked all the returns and cannot see any data except message returned? 
The goroutine added in receive message is nice to avoid if not used....

Reply:
Its used by the `sender-destinations` request. That is the only one so far. 

I added the goroutine because previously each request was blocking, so if it took a `sender` 2 seconds to deliver a message. Then the whole server would be blocked for that time.

Reply:
Ah now i see it. But doesnt we break the design pattern by doing 2 way communication like this? 
I dont have all the code in my head at the momens (6 months since i looked at it...)

Reply:
I don't think so. In this case I didn't find an other solution because the result of a manual trigger of a destination is only relevant for the user that triggered it.

Yea I know! Took me a while to get going writing it as well :)

Anyhow, its probably pretty easy to refactor this later if we figure out a better way to do it